src/components/loaderScreen/index.js

```diff
@@ -44,18 +47,31 @@ export default class LoaderScreen extends BaseComponent {
   };
 
   render() {
-    const {message, messageStyle, loaderColor, overlay, backgroundColor, containerStyle, ...others} = this.props;
+    const {
+      message,
+      messageStyle,
+      loaderColor,
+      overlay,
+      backgroundColor,
+      customLoader,
+      containerStyle,
+      ...others
+    } = this.props;
 
     return (
       <View style={[overlay ? [styles.overlayContainer, {backgroundColor}] : styles.container, containerStyle]}>
         <View flex center>
-          <ActivityIndicator
-            size={'large'}
-            animating
-            color={loaderColor || (Constants.isIOS ? Colors.dark60 : Colors.primary)}
-            {...others}
-          />
-          {!overlay && message && <Text style={[styles.message, messageStyle]}>{message}</Text>}
+          {customLoader ? (
+            <View>{customLoader}</View>
```

Comment:
Why do we need to wrap the 'customLoader' with a View?

Reply:
We don't.. 
Fixed :)